example/.env-example

```diff
@@ -0,0 +1 @@
+AFRICASTALKING_API_KEY = 235ded9cc0847a38554jnbv74s8fcs5c5dwt5ew521r5ws5r5v524gv
```

Comment:
I don't think this is ok IMO, the docs quite asks you to replace this

Reply:
Yeah, I get your case. but I ain't interfering with the logic, However, if you technically look at my logic, we're simply threadsafing the environment variables, in this case, the AFRICASTALKING_API_KEY variable.

Please let me know what need to be changed or so?